lib/app/sequences/01_conformance_sequence.rb

```diff
@@ -173,7 +173,7 @@ class ConformanceSequence < SequenceBase
         assert (token_url =~ /\A#{URI::regexp(['http', 'https'])}\z/) == 0, "Invalid token url: '#{token_url}'"
 
         warning {
-          service = @conformance.try(:rest).try(:first).try(:security).try(:service).try(:coding).try(:code)
+          service = @conformance.try(:rest).try(:first).try(:security).try(:service).try(:first).try(:coding).try(:first).try(:code)
```

Comment:
These "first"s aren't precisely correct; if you come across a server that supports more than one security protocol, suddenly there will be an order dependency whereas the fhir specification would consider the order of multiple entries irrelevant.

Perhaps the philosophy is just to wait until it matters for a real user and fix it then (which is fine -- just want to make sure it is an intentional choice).

Reply:
Updated to check all security services, so it will work with conformance statements with multiple entries:

<img width="742" alt="screen shot 2018-10-03 at 11 10 10 am" src="https://user-images.githubusercontent.com/41651655/46419918-f1778400-c6fc-11e8-80ae-91db2a68d800.png">

Updated error messages to show what was found (for example with an accidental misspelling)

<img width="786" alt="screen shot 2018-10-03 at 11 12 23 am" src="https://user-images.githubusercontent.com/41651655/46420071-3e5b5a80-c6fd-11e8-99be-a36a95b45fa1.png">